packages/viewer/src/lib/scenes/scene.ts

```diff
@@ -559,17 +569,139 @@ export class SceneItemOperationsBuilder
   }
 }
 
+/**
+ * A class that is responsible for building operations on pmi annotations for a specific scene.
+ * This executor requires a query, and expects `execute()` to be invoked in
+ * order for the changes to take effect.
+ */
+export class PmiAnnotationOperationsBuilder
+  implements
+    PmiAnnotationOperations<PmiAnnotationOperationsBuilder>,
+    SceneElementOperationsBuilder
+{
+  private builder: PmiAnnotationOperationBuilder;
+
+  public constructor(
+    private query: QueryExpression,
+    givenBuilder?: PmiAnnotationOperationBuilder
+  ) {
+    this.builder = givenBuilder ?? new PmiAnnotationOperationBuilder();
+  }
+
+  public isItemBuilder(): this is SceneItemOperationsBuilder {
+    return false;
+  }
+
+  public isAnnotationBuilder(): this is PmiAnnotationOperationsBuilder {
+    return true;
+  }
+
+  /**
+   * Specifies that the PMI annotations matching the query should be hidden.
+   *
+   * @example
+   * ```typescript
+   * const viewer = document.querySelector('vertex-viewer');
+   * const scene = await viewer.scene();
+   *
+   * // Hide the item with the `item-uuid` ID
+   * await scene.elements((op) => [
+   *   op.items.where((q) => q.withItemId('item-uuid')).hide(),
+   * ]).execute();
```

Comment:
Looks like these examples still reference the `items` equivalent, could we update them to the `annotations` operation?

Reply:
Good catch, updated!